### PR TITLE
lll: Advertise max line length instead of just reporting failure

### PR DESCRIPTION
**Pull request by blast-hardcheese**
Pardon if this has been discussed before, I couldn't find it in the issue/PR list.

As a user, I find it disorienting when tools inform me that I have made a mistake but do not offer any insights into how to resolve the issue.

In this case in particular, we have all the information, but then require the user to know the defaults and to traverse the config overrides.

It'd be nice to just provide this information. Impact to existing users should be minimal, including data transfer or storage of logs in CI, since teams who are already in compliance would not get the new log component, and teams who are not in compliance yet would get the message once and then be done.

Happy to circle back if there's prior art for how this information should be displayed.

Thanks!

### CODE REVIEW

**Comment by ldez** [comment]
Hello, it's a good suggestion, I rephrased the message to be more explicit.

File: pkg/golinters/lll/lll.go [hunk 1]
```diff
@@ -121,7 +121,7 @@ func getLLLIssuesForFile(filename string, maxLineLen int, tabSpaces string) ([]r
 					Filename: filename,
 					Line:     lineNumber,
 				},
-				Text:       fmt.Sprintf("line is %d characters", lineLen),
+				Text:       fmt.Sprintf("The line is %d characters long, which exceeds the maximum of %d characters.", lineLen, maxLineLen),
```
Comment (Antonboom) [inline pkg/golinters/lll/lll.go:124]: Usually linters warnings not ended with period?

**Review by Antonboom — APPROVED** [review]

File: pkg/golinters/lll/lll.go (hunk 1 above)
Comment (Antonboom) [reply pkg/golinters/lll/lll.go:124]: And not capitalized 🤔

Comment (ldez) [reply pkg/golinters/lll/lll.go:124]: I think there is no convention about linter messages: some have a period, others do not, and the same thing applies to capitalization.

**Review by ldez — APPROVED** [review]

**Pull request merged**